### ISSUE TITLE
Adding location manually under "Locations" module is not adding any location and showing status as "Fail".

### ISSUE THREAD

**Issue by onepointpb** [OnePointGlobal/OnePoint-Global-Account-Toolkit#438]
Adding location manually under "Locations" module is not adding any location and showing status as "Fail".
![image](https://user-images.githubusercontent.com/26476815/50625916-1922f600-0f51-11e9-894d-a79c3603bbd8.png)

**Comment by dougmorris** [comment]
@brijeshonepoint exactly why is this a server side issue? Can you be more specific please.

**Comment by brijeshonepoint** [comment]
> 
> 
> @brijeshonepoint exactly why is this a server side issue? Can you be more specific please.

Yes @dougmorris , I am getting status key as Fail in the response which is showing in the page. Below is the request and response.

Request:
{"Data":"{\"AddressName\":\"btm 1st stage\",\"AddresslistId\":\"2606\"}"}

Response:
[{"Distance":0.0,"SurveyID":0,"SurveyReference":null,"Range":0,"SurveyName":null,"CreatedDateForJson":"05/28/2019","EncodeAddress":"btm 1st stage","AddressID":5308,"AddressListID":2606,"Address":"btm 1st stage","Latitude":0.0,"Longitude":0.0,"GeoCode":"",**"Status":"Fail",**"IsDeleted":false,"UserDef1":"","UserDef2":"","UserDef3":"","UserDef4":"","UserDef5":"","UserDef6":"","UserDef7":"","UserDef8":"","UserDef9":"","CreatedDate":"2019-05-28T05:06:05.225244Z","LastUpdatedDate":"2019-05-28T05:06:05.225244Z"}]

**Comment by brijeshonepoint** [comment]
Issue is with Latitude and longitude value which was sent as 0, since auto complete drop-down is not working we are facing this issue. If I am manually entering some value it is working and sending status as success.
Need to integrate with map quest to set values automatically. 
#424 is the same issue we can close one of them, I will implement map quest to get values from the list.
